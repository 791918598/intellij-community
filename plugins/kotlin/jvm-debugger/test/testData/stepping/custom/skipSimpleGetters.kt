fun foo(name: String, age: Int): Any {
    return name
}

class User1(
    val name: String,
    val age: Int
)

private fun test1() {
    val user = User1("", 0)
    // STEP_INTO: 1
    // RESUME: 1
    //Breakpoint!
    foo(user.name, user.age)
}

class User2(val name: String, val age: Int)

private fun test2() {
    val user = User2("", 0)
    // STEP_INTO: 1
    // RESUME: 1
    //Breakpoint!
    foo(user.name, user.age)
}

fun main() {
    test1()
    test2()
}

// SKIP_GETTERS: true
