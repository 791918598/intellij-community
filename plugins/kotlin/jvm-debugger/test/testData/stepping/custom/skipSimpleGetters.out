LineBreakpoint created at skipSimpleGetters.kt:15
LineBreakpoint created at skipSimpleGetters.kt:25
Run Java
Connected to the target VM
skipSimpleGetters.kt:15
skipSimpleGetters.kt:2
skipSimpleGetters.kt:25
skipSimpleGetters.kt:2
Disconnected from the target VM

Process finished with exit code 0
